### PR TITLE
Upgrade AliCloud to provider standards

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ provider are installed by running `make ensure` in the root of the repository.
 
 ## Running Integration Tests
 
-The examples and integration tests in this repository will create and destroy real AliCloud
+The examples and integration tests in this repository will create and destroy real AWS
```
Comment (mikhailshilkov) [inline CONTRIBUTING.md]: This is probably not intended

File: COPYRIGHT [hunk 2]
```diff
@@ -5,7 +5,7 @@ LICENSE or <http://www.apache.org/licenses/LICENSE-2.0>).
 This project is a larger work that combines with software written
 by third parties, licensed under their own terms.
 
-Notably, this larger work combines with the Terraform AliCloud  Provider,
-which is licensed under the Mozilla Public License 2.0
-(see <https://www.mozilla.org/en-US/MPL/2.0/> or the project itself at
-<https://github.com/terraform-providers/terraform-provider-alicloud>).
+Notably, this larger work combines with the Terraform AWS Provider,
```
Comment (mikhailshilkov) [inline COPYRIGHT]: The changes in this file are probably not intended

File: resources.go [hunk 3]
```diff
@@ -28,54 +29,85 @@ const (
...
+	actionTrailMod   = "ActionTrail"
+	aliKafaMod       = "AliKafka"
+	apiGatewayMod    = "ApiGateway"
+	casMod           = "Cas"
+	cdnMod           = "Cdn"
+	cenMod           = "Cen"
+	cloudConnectMod  = "CloudConnect"
+	cmsMod           = "Cms"
+	crMod            = "Cr"
+	csMod            = "Cs"
```
Comment (mikhailshilkov) [inline resources.go]: Should these be `CR` and `CS`?

File: resources.go [hunk 4]
```diff
@@ -28,54 +29,85 @@ const (
...
+	datahubMod       = "Datahub"
+	ddsMod           = "Dds"
+	ddosMod          = "Ddos"
+	dnsMod           = "Dns"
+	drdsMod          = "Drds"
+	ecsMod           = "Ecs"
+	elasticsearchMod = "ElasticSearch"
+	emrMod           = "Emr"
+	essMod           = "Ess"
+	fcMod            = "Fc"
```
Comment (mikhailshilkov) [inline resources.go]: Should this be `FC`?

File: CONTRIBUTING.md (hunk 1 above)
Comment (stack72) [reply CONTRIBUTING.md]: fixed :)

File: resources.go [hunk 5]
```diff
@@ -28,54 +29,85 @@ const (
...
+	gpdbMod          = "Gpdb"
+	kmsMod           = "Kms"
+	kvstoreMod       = "KVStore"
+	logMod           = "Log"
+	marketPlaceMod   = "MarketPlace"
+	mongoDbMod       = "MongoDb"
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
	mongoDbMod       = "MongoDB"
```

File: resources.go [hunk 6]
```diff
@@ -28,54 +29,85 @@ const (
...
+	mnsMod           = "Mns"
+	nasMod           = "Nas"
+	ossMod           = "Oss"
+	otsMod           = "Ots"
+	polarDbMod       = "PolarDb"
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
	polarDbMod       = "PolarDB"
```

File: resources.go [hunk 7]
```diff
@@ -28,54 +29,85 @@ const (
...
+	pvtzMod          = "Pvtz"
+	ramMod           = "Ram"
+	rocketMqMod      = "RocketMq"
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
	rocketMqMod      = "RocketMQ"
```

File: resources.go [hunk 8]
```diff
@@ -28,54 +29,85 @@ const (
...
+	vpcMod           = "Vpc"
+	vpnMod           = "Vpn"
+	yundunMod        = "Yundun"
 )
 
-// alicloudMember manufactures a type token for the AliCloud package and the given module and type.
-func alicloudMember(mod string, mem string) tokens.ModuleMember {
-	return tokens.ModuleMember(alicloudPkg + ":" + mod + ":" + mem)
+var namespaceMap = map[string]string{
+	"alicloud": "Alicloud",
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
	"alicloud": "AliCloud",
```

File: resources.go [hunk 9]
```diff
@@ -90,170 +122,584 @@ func Provider() tfbridge.ProviderInfo {
...
+				Docs: &tfbridge.DocInfo{
+					Source: "alicloud_ess_lifecycle_hook",
+				},
+			},
+			"alicloud_ess_scaling_configuration":       {Tok: resource(essMod, "ScalingConfiguration")},
+			"alicloud_ess_scaling_group":               {Tok: resource(essMod, "ScalingGroup")},
+			"alicloud_ess_scaling_rule":                {Tok: resource(essMod, "ScalingRule")},
+			"alicloud_ess_schedule":                    {Tok: resource(essMod, "Schedule")},
+			"alicloud_ess_notification":                {Tok: resource(essMod, "Notification")},
+			"alicloud_ess_scalinggroup_vserver_groups": {Tok: resource(essMod, "ScalingGroupVserverGroups")},
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
			"alicloud_ess_scalinggroup_vserver_groups": {Tok: resource(essMod, "ScalingGroupVServerGroups")},
```

File: resources.go [hunk 10]
```diff
@@ -90,170 +122,584 @@ func Provider() tfbridge.ProviderInfo {
...
+				},
+			},
+			"alicloud_route_entry":                         {Tok: resource(vpcMod, "RouteEntry")},
+			"alicloud_route_table":                         {Tok: resource(vpcMod, "RouteTable")},
+			"alicloud_route_table_attachment":              {Tok: resource(vpcMod, "RouteTableAttachment")},
+			"alicloud_router_interface":                    {Tok: resource(vpcMod, "RouterInterface")},
+			"alicloud_router_interface_connection":         {Tok: resource(vpcMod, "RouterInterfaceConnection")},
+			"alicloud_nat_gateway":                         {Tok: resource(vpcMod, "NatGateway")},
+			"alicloud_forward_entry":                       {Tok: resource(vpcMod, "ForwardEntry")},
+			"alicloud_havip":                               {Tok: resource(vpcMod, "HaVip")},
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
			"alicloud_havip":                               {Tok: resource(vpcMod, "HAVip")},
```

File: resources.go [hunk 11]
```diff
@@ -90,170 +122,584 @@ func Provider() tfbridge.ProviderInfo {
...
+			"alicloud_havip_attachment":                    {Tok: resource(vpcMod, "HaVipAttachment")},
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
			"alicloud_havip_attachment":                    {Tok: resource(vpcMod, "HAVipAttachment")},
```

File: resources.go [hunk 12]
```diff
@@ -90,170 +122,584 @@ func Provider() tfbridge.ProviderInfo {
...
+			"alicloud_ssl_vpn_client_cert":  {Tok: resource(vpnMod, "SslVpnClientCert")},
+			"alicloud_ssl_vpn_server":       {Tok: resource(vpnMod, "SslVpnServer")},
+			"alicloud_vpn_connection":       {Tok: resource(vpnMod, "Connection")},
+			"alicloud_vpn_customer_gateway": {Tok: resource(vpnMod, "CustomerGateway")},
+			"alicloud_vpn_gateway":          {Tok: resource(vpnMod, "Gateway")},
+			"alicloud_vpn_route_entry":      {Tok: resource(vpnMod, "RouteEntry")},
+
+			// Yundun
+			"alicloud_yundun_bastionhost_instance": {Tok: resource(yundunMod, "BastionHostInstance")},
+			"alicloud_yundun_dbaudit_instance":     {Tok: resource(yundunMod, "DbAuditInstance")},
```
Comment (mikhailshilkov) [inline resources.go]:
```suggestion
			"alicloud_yundun_dbaudit_instance":     {Tok: resource(yundunMod, "DBAuditInstance")},
```

**Comment by stack72** [comment]
@mikhailshilkov all yours again :) Thanks for picking up all of those things before we publish it!

**Pull request merged**